Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -320,6 +324,24 @@ private extension AppSettingsStore {
         }
         onCompletion(numberOfDaysSinceLastDismissal >= 5)
     }
+
+    /// Sets the EU Shipping Notice dismissal state into `GeneralAppSettings`
+    ///
+    func setEUShippingNoticeDismissState(isDismissed: Bool, onCompletion: (Result<Void, Error>) -> Void) {
+        do {
+            try generalAppSettings.setValue(isDismissed, for: \.isEUShippingNoticeDismissed)
```

Comment:
Since we only ever dismiss this notice (we don't set it back to `true` to make it visible again), is it worth simplifying this to something like `dismissEUShippingNotice` or `setEUShippingNoticeDismissed` without taking the `isDismissed` parameter?

Reply:
Same fix as in https://github.com/woocommerce/woocommerce-ios/pull/9695#discussion_r1191761873. Thanks!